src/Symfony/Bundle/FrameworkBundle/Tests/CacheWarmer/AnnotationsCacheWarmerTest.php

```diff
@@ -166,10 +166,7 @@ public function testWarmupRemoveCacheMisses()
         $this->assertTrue(isset($data[0]['bar_hit']));
     }
 
-    /**
-     * @return MockObject|Reader
```

Comment:
all of those mocks should be `MockObject&Reader`. so removing them would hurt us when upgrading to intersection types.

Reply:
Right! Fixed in #42216